pkgs/by-name/ce/celeste64/package.nix

```
@@ -0,0 +1,61 @@
+{
+  lib,
+  buildDotnetModule,
+  dotnetCorePackages,
+  fetchFromGitHub,
+  SDL2,
+  libGL,
+  fmodex,
+  libdecor,
+  libxkbcommon,
+  wayland,
+  systemd,
+  alsa-lib,
+  libselinux,
+  libX11,
+  withSELinux ? false,
+  ...
+}:
+buildDotnetModule {
+  pname = "celeste64";
+  version = "unstable-2024.01.31";
```

```suggestion
  version = "0-unstable-2024.01.31";
```

Please see https://github.com/NixOS/nixpkgs/pull/285232#discussion_r1472515445 and #280312